beetsplug/mbsync.py

```diff
@@ -109,69 +122,86 @@ def albums(self, lib, query, move, pretend, write):
 
             items = list(a.items())
 
-            # Get the MusicBrainz album information.
-            album_info = hooks.album_for_mbid(a.mb_albumid)
-            if not album_info:
-                self._log.info(u'Release ID {0} not found for album {1}',
-                               a.mb_albumid,
-                               album_formatted)
-                continue
+            valid_albumid = re.match(
+                     r"(\d|\w){8}-(\d|\w){4}-(\d|\w){4}-(\d|\w){4}-(\d|\w){12}",
```

Comment:
Also, deduplicating the regex so it doesn’t need to be copied & pasted would be awesome.